### ISSUE TITLE
[Bug Report] 2.3.0-beta.1 - Can't resolve 'ress/dist/ress.min.css'

### ISSUE THREAD

**Issue by jacekkarczmarczyk** [vuetifyjs/vuetify#11276]
### Environment
**Vuetify Version:** 2.3.0-beta.1
**Last working version:** 2.3.0-beta.0
**Vue Version:** 2.6.11
**Browsers:** Chrome 81.0.4044.122
**OS:** Windows 10

### Steps to reproduce
upgrade to 2.3.0-beta.1
build/serve

### Expected Behavior
project compiles

### Actual Behavior
```
 ERROR  Failed to compile with 1 errors9:42:08

 error  in ./node_modules/vuetify/src/styles/main.sass

Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleNotFoundError: Module not found: Error: Can't resolve 'ress/dist/ress.min.css' in 'C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\vuetify\src\styles'
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\webpack\lib\Compilation.js:925:10
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\webpack\lib\NormalModuleFactory.js:401:22
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\webpack\lib\NormalModuleFactory.js:130:21
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\webpack\lib\NormalModuleFactory.js:224:22
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\neo-async\async.js:2830:7
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\neo-async\async.js:6877:13
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\webpack\lib\NormalModuleFactory.js:214:25
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\enhanced-resolve\lib\Resolver.js:213:14
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\enhanced-resolve\lib\Resolver.js:285:5
    at eval (eval at create (C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:15:1)
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:44:7
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\enhanced-resolve\lib\Resolver.js:285:5
    at eval (eval at create (C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:15:1)
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\enhanced-resolve\lib\Resolver.js:285:5
    at eval (eval at create (C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:27:1)
    at C:\cygwin64\home\PC\jacekkarczmarczyk\rachunek\node_modules\enhanced-resolve\lib\DescriptionFilePlugin.js:67:43

 @ ./node_modules/vuetify/lib/presets/default/index.js 2:0-39
 @ ./node_modules/vuetify/lib/services/presets/index.js
 @ ./node_modules/vuetify/lib/services/index.js
 @ ./node_modules/vuetify/lib/framework.js
 @ ./node_modules/vuetify/lib/index.js
 @ ./src/main.ts
 @ multi ./src/main.ts
```

### Reproduction Link
https://github.com/jacekkarczmarczyk/rachunek

### Other comments
Broken in https://github.com/vuetifyjs/vuetify/commit/f70de6036fa0136f72d5d88471d736b621407875

**Comment by johnleider** [comment]
This works as normal on W10, need more info.

**Comment by jacekkarczmarczyk** [comment]
updated repro to use 2.3.0-beta.1

**Comment by jacekkarczmarczyk** [comment]
Fixed in #11277
